components/ProposalRow.tsx

```diff
@@ -34,6 +38,27 @@ export default function ProposalRow({
     args: [id],
     value: fees?.l1 || BigInt(0),
   });
+  const bridgeButton =
+    bridged !== null && !proposalMetadataBridgeIsLoading ? (
+      <div className="rounded bg-gray-500 px-2 py-1 text-xs font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
+        Waiting for relayer 🕓
+      </div>
+    ) : (
+      <button
+        className="rounded bg-indigo-600 px-2 py-1 text-xs font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+        onClick={async () => {
+          await walletClient!.switchChain({ id: l1.chain.id });
+          bridgeProposal();
+        }}
+        disabled={!bridgeProposal || bridgeProposalIsLoading}
```

Comment:
For style consistency, we want to disable the button and make it look gray when it's clicked and pending.
<img width="207" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/c72e5da7-5996-432d-bbfb-590d6fd1ab3c">
Like this:
<img width="228" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/91facb23-0da9-4b87-b4f3-edae4445ae98">